src/components/DependencyDetailsTable/DependencyDetailsTable.js

```diff
@@ -40,7 +41,13 @@ class DependencyDetailsTable extends Component<Props> {
         <tbody>
           <tr>
             <FirstCell>
-              <Label>Location</Label>
+              <Label>
+                Location
+                <HelpButton
+                  size={16}
+                  href="https://github.com/joshwcomeau/guppy/blob/dependenciesExplanationDocs/docs/understanding-dependencies.md#dependencies-vs-devdependencies"
```

Comment:
I think you can update this pre-emptively to:

```
https://github.com/joshwcomeau/guppy/blob/master/docs/understanding-dependencies.md#dependencies-vs-devdependencies
```

The link won't work until it's merged, but I checked the existing docs and this URL structure should work :)